api/tests/integration/infrastructure/repositories/organization-for-admin-repository_test.js

```diff
@@ -154,4 +154,175 @@ describe('Integration | Repository | Organization-for-admin', function () {
       });
     });
   });
+
+  describe('#save', function () {
+    it('should update invitation status for given organization', async function () {
```

Comment:
suggestion : mentionner qu'on passe d'un état pending à cancelled ? Vu qu'on ne passe pas des variables en paramètre pour définir un état 1 et un état 2, je serais d'avis de rendre le test plus explicite 
`should update pending invitation status to cancelled status for given organization`

Reply:
change :)